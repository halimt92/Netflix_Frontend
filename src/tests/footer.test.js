import React from 'react';
import { shallow } from 'enzyme';
import {
  ImFacebook2, ImInstagram, ImTwitter, ImYoutube,
} from 'react-icons/im';
import Footer from '../components/footer/Footer';

describe('Footer', () => {
  it('should render a facebook icon', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(ImInstagram)).toHaveLength(1);
  });

  it('should render a instagram icon', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(ImFacebook2)).toHaveLength(1);
  });

  it('should render a Twitter icon', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(ImTwitter)).toHaveLength(1);
  });

  it('should render a YouTube icon', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find(ImYoutube)).toHaveLength(1);
  });

  it('matches Footer snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
