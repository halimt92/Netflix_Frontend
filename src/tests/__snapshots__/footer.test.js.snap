// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches Footer snapshot 1`] = `ShallowWrapper {}`;
